### PR TITLE
feat:-add authenticatio page

### PR DESCRIPTION
**Pull request by shubhamkumar648**
authentication part
1. add login
2. signup
login
login apage
signup

### CODE REVIEW

**Review by vaibhav-mahalle — APPROVED** [review]
- You could have created separate branches for login and sign up.
- not major changes required.
- Looking good.

**Review by krishanu2911 — APPROVED** [review]
no changes required everything seems good to go nice design

**Comment by shubhamkumar648** [comment]
> * You could have created separate branches for login and sign up.
> * not major changes required.
> * Looking good.

ok I will do thank you

**Comment by shubhamkumar648** [comment]
> no changes required everything seems good to go nice design

Thank you

File: stylesheets/login.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+.inp_area {
+    padding: 0.7rem;
+    width: 100%;
+    border-radius: 5px;
+    font-size: 1rem;
+    border: 1px solid var(--dark-grey);
+}
+
+.login_color {
+
```
Comment (sakleprajwal) [inline stylesheets/login.css]: Avoid unnecessary spaces

**Review by sakleprajwal — APPROVED** [review]
Suggested a minor change. Everything else looks good.

File: stylesheets/login.css (hunk 1 above)
Comment (shubhamkumar648) [reply stylesheets/login.css]: will do, thank you

**Pull request merged**